zerver/openapi/zulip.yaml

```diff
@@ -14059,6 +14059,21 @@ paths:
                     - 2
                     - 3
                   example: 1
+                week_start_day:
+                  description: |
+                    First day of the week used by date pickers in the web/desktop application.
+
+                    - 1 - Automatic (follows system settings)
+                    - 2 - Saturday
+                    - 3 - Sunday
+                    - 4 - Monday
+                  type: integer
+                  enum:
+                    - 1
+                    - 2
+                    - 3
+                    - 4
+                  example: 1
```

Comment:
This would need a "Changes" entry and a corresponding entry in changelog as well. Please go through https://zulip.readthedocs.io/en/latest/documentation/api.html